src/common/interceptors/response.interceptor.spec.ts

```
@@ -0,0 +1,158 @@
+      switchToHttp: jest.fn().mockReturnValue({
+        getResponse: jest.fn().mockReturnValue({
+          locals: {},
+        }),
+      }),
+    } as any;
+
+    // Mock CallHandler
+    mockCallHandler = {
+      handle: jest.fn(),
+    } as any;
+  });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)